app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './AdminEmail.module.scss';
+import CopyToClipboard from '../../../../Components/CopyToClipboard/CopyToClipboard';
```

Comment:
can we use the absolute path here?
```suggestion
import CopyToClipboard from 'Components/CopyToClipboard/CopyToClipboard';
```

Reply:
Done